Themes/default/css/index.css

```diff
@@ -7,14 +7,12 @@ body {
 	background: #e9eef2;
 	font: 83.33%/150% "Segoe UI", "Helvetica Neue", "Nimbus Sans L", Arial, "Liberation Sans", sans-serif;
 	color: #4d4d4d;
-	margin: 0;
-	padding: 0;
 	display: flex;
 	flex-direction: column;
 	min-height: 100vh;
 }
-/* <p> is a possible exception for margins. Must test that one a bit. */
-p, ul, ol, li, dl, dd ,dt, fieldset, form {
+/* General reset */
+* {
```

Comment:
Margin/padding should be fine, browser default ones are too inconsistent to be relied onto. Border-box could cause some harm, but at most it'll break somewhere that widths are used that conflict with padding, better to find those early and they're very easy to fix. I looked at a wide range of pages and didn't find any problems so far, other than the two already fixed.